src/runtime.js

```diff
@@ -21,6 +21,10 @@ var Compiletime = {
 // code used both at compile time and runtime is defined here, then put on
 // the Runtime object for compile time and support.js for the generated code
 
+function getPointerSize() {
+  return MEMORY64 ? 8 : 4;
+}
+
```

Comment:
I think this can be removed and just use `POINTER_SIZE`.

Reply:
It looks like parseTools is executed first in compiler.js. So we could define `POINTER_SIZE` there and use it here. I think we already define it there, so maybe after recent changes it will just work?

If not, what error do you get?

Reply:
I could go figure that all out, but I'd rather do this in a follow-up.